src/hooks/govwallet/useGovWalletBalance/useGovWalletBalance.tsx

```diff
@@ -0,0 +1,81 @@
+import { useCallback, useContext, useEffect, useState } from "react";
+import { CampaignConfigContext } from "../../../context/campaignConfig";
+import { getGovWalletBalance } from "../../../services/govwallet/balance";
+
+export type GovWalletBalanceState = "DEFAULT" | "SUFFICIENT" | "INSUFFICIENT";
```

Comment:
Because the condition is exactly equal to 10000, do you think changing this to eligible or ineligible would be better? Since 10001 is technically "sufficient"

Reply:
Right, that makes sense, renamed `SUFFICIENT` to `ELIGIBLE`, and `INSUFFICIENT` to `INELIGIBLE` in [e4aeff8](https://github.com/rationally-app/mobile-application/pull/606/commits/e4aeff8d70c5f173d63d6cf25e700e3118e56a7e)!

Reply:
Thanks!